config/webpacker.yml

```diff
@@ -86,28 +89,11 @@ test:
   # Compile test packs to a separate directory
   public_output_path: packs-test
 
-
-staging:
```

Comment:
Does the removal of the `staging` section mean that the production config is automatically picked on the staging server?
Or does that need a configuration change on the staging server (setting `NODE_ENV=production` for example)?
(I'm in favor of removing the staging settings btw, I like our staging environment to be as close as possible to production)

Reply:
I'm not sure, I just adapted the default webpacker 6 config. Guess I missed the change. I think that the node env (maybe rails env too?) may have to be changed on the server from staging to production to make it work, but I don't know for sure.

Reply:
We'll see when we deploy 🤷